src/vm/class.cpp

```diff
@@ -1664,6 +1664,31 @@ CorElementType MethodTable::GetHFAType()
         _ASSERTE(pMT->IsValueType());
```

Comment:
Do you need to modify all callers of GetHFAType to handle these new values correctly?

Reply:
I thought I looked at them all,but maybe I maybe I waiting for an assert to fire...  I'll take a look next week.